storage/asynchbase/src/test/java/net/opentsdb/storage/MockBase.java

```diff
@@ -1598,8 +1614,12 @@ public String toString() {
 
         if (family == null || family.length < 1) {
           for (final Entry<byte[], ByteMap<ByteMap<TreeMap<Long, byte[]>>>> cf : map) {
-            final Iterator<Entry<byte[], ByteMap<TreeMap<Long, byte[]>>>>
-            cursor = cf.getValue().iterator();
+            final Iterator<Entry<byte[], ByteMap<TreeMap<Long, byte[]>>>> cursor;
+            if (is_reversed) {
```

Comment:
What is this for?

Reply:
for scanning in reverse starting with the most recent data. E.g. like Splunk we'll scan in chunks, returning the most recent first then iterating backwards in time.